packages/odyssey-react/src/components/Text/index.tsx

```diff
@@ -0,0 +1,167 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import type {
+  FunctionComponent,
+  ReactText,
+  ReactNode,
+  HTMLAttributes,
+} from "react";
+import { useCx, useOmit } from "../../utils";
+import styles from "./Text.module.scss";
+import ScreenReaderText from "../ScreenReaderText";
+
+export interface Props extends HTMLAttributes<HTMLElement> {
+  /**
+   * Text content to be rendered
+   */
+  children: ReactText | ReactNode;
+
+  /**
+   * Prepends visually hidden assistive text to the child contents
+   */
+  a11yStart?: ReactText;
+
+  /**
+   * Appends visually hidden assistive text to the child contents
+   */
+  a11yEnd?: ReactText;
```

Comment:
hmmm, this feels like an example where we should open the API to composition rather than adding specific placeholders where prop values are injected into `<ScreenReaderText>`. A consumer of Odyssey should be able to place `<ScreenReaderText>` anywhere within a `<Text>` component

Reply:
Good feedback, I have updated.